src/UBC Toboggan/Assets/Scripts/scoreManager.cs

```diff
@@ -0,0 +1,83 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class scoreManager : MonoBehaviour
+{
+    public static float score = 0f;
+    
+    public Text scoreText;
+    public GameObject flipBonusText;
+    public GameObject airBonusText;
+    public GameObject player;
+
+    public float flipBonus = 5f;
+    public float airTimeMultiplier = 1f;
+
+    float flipScore = 0f;
+    float airScore = 0f;
+    bool showingBonus = false;
+
+    playerManager playerManagerScript;
+
+    void Start() {
+        flipBonusText.SetActive(false);
+        airBonusText.SetActive(false);
+
+        score = 0;
+
+        playerManagerScript = player.GetComponent<playerManager>();
+    }
+
+    void Update() {
+        scoreText.text = score.ToString();
+
+        // begin showing score text when a point is gained in the air
+        if (!playerManagerScript.grounded && !showingBonus && (flipScore > 0f || airScore > 0f)) {
+            updateFlipText();
+            updateAirText();
```

Comment:
In `playerManagerScript`'s `Update()`, `addFlipScore()` and `addAirTimeScore()` are called and those methods call `updateFlipText()` and `updateAirText()` respectively. Therefore, do we need these calls here?